File: cdn-load-balancer/android/ExoPlayerCDNLoadBalancerJava/app/src/main/java/io/streamroot/lumen/delivery/client/samples/cdn_load_balancer/exoplayer/Application.java
package io.streamroot.lumen.delivery.client.samples.cdn_load_balancer.exoplayer;

import androidx.multidex.MultiDexApplication;

import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient;

public final class Application extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        LumenDeliveryClient.initializeApp(this);
    }
}

File: cdn-load-balancer/android/ExoPlayerCDNLoadBalancer/app/src/main/java/io/streamroot/lumen/delivery/client/samples/cdn_load_balancer/exoplayer/Application.kt
package io.streamroot.lumen.delivery.client.samples.cdn_load_balancer.exoplayer

import androidx.multidex.MultiDexApplication
import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient

class Application : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        LumenDeliveryClient.initializeApp(this)
    }
}